### ISSUE TITLE
error in ggsave and to_inch for mm

### ISSUE THREAD

**Issue by janschulz** [yhat/ggplot#431]
the to_inch for mm is wrong, it should be 

```
to_inch = {"in":lambda x:x,"cm":lambda x: x / 2.54, "mm":lambda x: x / (2.54 * 10)}
assert (to_inch["mm"](2) - 0.0787402) < 0.001 # from google "2mm to inch"
```

**Comment by has2k1** [comment]
Yeah, I fixed that for #360. I think it it was not helped by the coding style i.e multiple entries on a single line. It makes it difficult to see at a glance what is going on.

**Comment by glamp** [comment]
fixed in 0.9.3
